src/Definitions/RandomCocktail.ts

```diff
@@ -0,0 +1,55 @@
+export interface RandomCocktail {
```

Comment:
Created this file within the new definitions folder that way our code looks cleaner.

Reply:
Is this done per API that we have?